.github/PULL_REQUEST_TEMPLATE/mitigation-solution.md

```diff
@@ -0,0 +1,8 @@
+**This solution referes to which of the apps?**
+A# - App's name here
+
+**What did you do to mitigate the vulnerability?**
+A clear and concise description of what you did. 
+
+**Did you test your changes? What commands did you run?**
+Yes. First I used the command ....
```

Comment:
Maybe add `A good idea is trying to reproduce the attack narrative. If it doesn't work, congratulations, you found a valid mitigation!` ?

Reply:
That's great! Thanks, @Takehime!